packages/lesswrong/components/questions/AnswersSorting.tsx

```diff
@@ -0,0 +1,77 @@
+import { registerComponent } from '../../lib/vulcan-lib';
```

Comment:
I mostly copy-pasted the `CommentsViews`, there is a bunch of duplication and it can probably be abstracted in a reusable component, but I don't know if it's worth it

Reply:
I would be on the fence about combining. Seems fine not to.